core/src/Streamly/Internal/Data/Stream/StreamD/Eliminate.hs

```diff
@@ -308,6 +326,12 @@ parseBreak (PRD.Parser pstep initial extract) stream@(Stream step state) = do
             PR.Error err ->
                 return (Left (ParseError err), StreamD.nil)
 
+-- | Parse a stream using the supplied 'Parser'.
+--
+{-# INLINE parseBreak #-}
+parseBreak :: Monad m => PR.Parser a m b -> Stream m a -> m (Either ParseError b, Stream m a)
```

Comment:
This is an internal module but the naming inconsistency will be changed later right?
`D` is default
`K` is something additional

Reply:
parseBreakD is using ParserD, parseBreak uses Parser. Both use StreamD.